res/Sectors/M1105/centrax.txt

```diff
@@ -0,0 +1,459 @@
+# Generated by https://travellermap.com
+# 2018-11-05T13:05:23+00:00
+
+# Centrax
+# 4,3
+
+# Name: Centrax (hv)
+
+# Milieu: M1105
+
+# Credits: Centrax sector was designed by Mark Humphreys. After dot-map by Leroy W. L. Guatney http://home.earthlink.net/~ngc5139/FDA/
+
+# Author:    Mark Humphreys
+
+# Subsector A: Kornaskereq
+# Subsector B: Slomua
+# Subsector C: Wrunoothon
+# Subsector D: Liana
+# Subsector E: Neerndantena
+# Subsector F: Konpeq
+# Subsector G: Wrilootanskuk
+# Subsector H: Aexun
+# Subsector I: Blexiontcar
+# Subsector J: Arephlaft
+# Subsector K: Uciennatin
+# Subsector L: Wruoruaft
+# Subsector M: Meexiannelizi
+# Subsector N: Araurnuansk
+# Subsector O: Pila
+# Subsector P: Wrogavaoqgior
+
+# Alleg: Cs: "Client state, Third Imperium"
```

Comment:
There really are 2 Imperial Client State worlds in Centrax?

Reply:
I'll check all of these as well, the CS default text gets pulled back from TravellerMap, it would be easier if I switched all of the Allegiances to be 4 character so there could be NaHv and CsHv (for example)

Reply:
A couple of sectors note that "Hc" is the two character Hiver Client State allegiance code.

Reply:
OK, I've changed the Cs ones to be Hc to be consistent